FreeRTOS/FreeRTOS
# PKCS #11 LTS sync

PKCS #11 LTS sync

Description
-----------
Copy over the code from release-candidate to the LTS development branch.

Test Steps
-----------
Run PKCS #11 demo.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
